src/encoded/static/components/series_search.js

```diff
@@ -0,0 +1,165 @@
+                setSeriesData(response);
+            });
+        }
+    };
+
+    // Select series from tab buttons
+    React.useEffect(() => {
+        handleClick(selectedSeries);
+    }, [handleClick, selectedSeries]);
+
+    /* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
+    return (
+        <div className="layout">
+            <div className="layout__block layout__block--100">
+                <div className="ricktextblock block series-search" data-pos="0,0,0">
+                    <h1>Functional genomics series</h1>
+                    <div className="outer-tab-container">
+                        <div className="tab-container series-tabs">
+                            {Object.keys(seriesList).map(s => (
+                                <button
+                                    key={s}
+                                    className={`tab-button${selectedSeries === s ? ' selected' : ''}`}
+                                    onClick={() => handleClick(s)}
+                                >
+                                    <div className="tab-inner">
+                                        <div className="tab-icon">
+                                            <img src={`/static/img/series/${s.replace('Series', '')}.svg`} alt={s} />
+                                        </div>
+                                        {seriesList[s].title}
+                                    </div>
+                                </button>
+                            ))}
+                        </div>
+                        <div className="tab-border" />
+                    </div>
+                    <div className="tab-body">
+                        <div className="tab-description">{descriptionData}</div>
+                        <div className="series-wrapper" onClick={e => handleLinks(e)} >
+                            <Panel>
+                                <PanelBody>
+                                    {seriesData ?
+                                        <ResultTable
+                                            context={seriesData}
+                                            searchBase={searchBase}
+                                            onChange={context.navigate}
+                                            currentRegion={currentRegion}
+                                            seriesFlag
+                                        />
+                                    : null}
+                                </PanelBody>
+                            </Panel>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    );
+    /* eslint-enable jsx-a11y/click-events-have-key-events */
+};
+
+SeriesSearch.contextTypes = {
+    location_href: PropTypes.string,
+    navigate: PropTypes.func,
+    fetch: PropTypes.func,
+};
+
+globals.contentViews.register(SeriesSearch, 'SeriesSearch');
```

Comment:
We would really need Keenan to confirm, but I feel like you could simplify this by making /series-search/ a real search view, adding to search_view.py. The advantage is that the returned JSON would know to go to /search-view/ instead of /search/ and you won’t have to have that `<div>` wrapper and click handler to modify the URL. So I’m thinking `/series-search/?type=OrganismDevelopmentSeries`. Each facet and tab click would just use links instead of GET requests. I could easily be missing something obvious that makes this impractical, but I feel like that would be simpler if it works.

Reply:
Updated.